src/components/kyb/BusinessInformationStep.tsx

```
@@ -0,0 +1,413 @@
+    }
+  };
+
+  const validateField = (field: string, value: string) => {
+    const newErrors = { ...errors };
+
+    switch (field) {
+      case 'legalCompanyName':
+        if (!value.trim()) {
+          newErrors.legalCompanyName = 'Legal company name is required';
+        } else {
+          delete newErrors.legalCompanyName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)